### PR TITLE
For #2052: Fix BrowserToolbar overlapping web content

### PR DESCRIPTION
**Pull request by jonalmeida**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt [hunk 1]
```diff
@@ -6,24 +6,50 @@ package org.mozilla.fenix.customtabs
...
+    engineView: View,
     onItemTapped: (ToolbarMenu.Item) -> Unit = {}
 ) : LifecycleAwareFeature, BackHandler {
 
     init {
+        // Remove toolbar shadow
         toolbar.elevation = 0f
+
+        // Reduce margin height of EngineView from the top for the toolbar
+        engineView.run {
```
Comment (ekager) [inline app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt]: is this working? I had to change the previous way to `swipeRefresh` to not crash

Comment (jonalmeida) [reply app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt]: Yes, the view name is probably inconsistent. I pass in the swipeRefresh ViewGroup to the integration class. I'll re-test to verify.

Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt]: Ohh I see its being passed in but is actually swipeRefresh, gotcha :)

Comment (jonalmeida) [reply app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt]: I've renamed it to `engineLayout` to fit the context it's in. 👍

**Pull request merged**